qml/pages/VenueList.qml

```diff
@@ -149,27 +149,16 @@ BVApp.Page {
                                    positionSource : page.positionSource
                                });
 
-                // v-play: TypeError: Property 'pushAttached' of object NavigationStack_QMLTYPE_59(0x7f9f0cb5ba90, "_NavigationStack") is not a function
-                var mapPage = pageStack.pushAttached(Qt.resolvedUrl("VenueMapPage.qml"),
+                // The icon parameter is only used on v-play
+                pageStack.pushAttached(Qt.resolvedUrl("VenueMapPage.qml"),
                                {
                                    venueCoordinate: QtPositioning.coordinate(currRestaurant.latCoord,
                                                                              currRestaurant.longCoord),
                                    positionSource: page.positionSource,
                                    name: currRestaurant.name
-                               });
-
-                // TODO: Directly assigning a coordinate property to Map.center seems to be broken
-                // with the current Qt version (5.2)
-                mapPage.map.addMapItem(mapPage.venueMarker)
-                mapPage.map.addMapItem(mapPage.currentPosition)
-
-                mapPage.setMapCenter(mapPage.venueMarker.coordinate)
-
-                // Seems to be broken with the current version of Qt :/
-                mapPage.map.fitViewportToMapItems()
+                               }, BVApp.Theme.iconBy("mapmarker")
```

Comment:
BTW: Regarding the chosen icon: I am very open for any suggestions. The v-play people have *so* many nice icons to choose from [in their IconType selection](https://v-play.net/doc/vplayapps-icontype/), Icons for Tux, icons for Apple, icons for XING, four (!!) icons for frickin' Facebook, but I couldn't find a single one, which properly symbolizes a map...